experiments/rtp-hdrext/playout-delay/index.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: playout-delay
+permalink: /experiments/rtp-hdrext/playout-delay/
+---
+
+Name: "Playout Delay" ; "RTP Header Extension to control Playout Delay"
+
+Formal name: <http://www.webrtc.org/experiments/rtp-hdrext/playout-delay>
+
+SDP "a= name": "playout-delay" ; this is also used in client/cloud signaling.
+
+Introduction
```

Comment:
Use markdown markup to indicate sections - I think it's "#" and "##" for first and second level headers.